signer.js

```diff
@@ -1,7 +1,9 @@
+const crypto = require('crypto');
+const elliptic = require('elliptic');
 const jsrsasign = require('jsrsasign');
```

Comment:
It is probably better to comment out the `require` and the actual code (JsrsasignSignatureSigner) not to load the script since it won't be really used.